client/jetpack-cloud/sections/partner-portal/hooks.tsx

```diff
@@ -477,3 +477,84 @@ export function useAssignMultipleLicenses(
 	const [ assign, isLoading ] = useAssignLicenses( selectedLicenseKeys, selectedSite );
 	return [ assign, isLoading ];
 }
+
+/**
+ * Returns product description and features with given product slug.
+ *
+ * @param productSlug
+ * @returns
+ */
+export function useProductDescription( productSlug: string ): {
+	description: TranslateResult | null;
+	features: ReadonlyArray< any >;
```

Comment:
Can we use a more specific type instead of `any` here? It looks like every feature is a string.

Reply:
Agree; I decided to use the `TranslateResult` type here since all feature is a translation strings. this is now fixed. 👍